01week/datatypes.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+
+//Write a JavaScript program to display the current day and time.
+
+const todaysDate = new Date; //declare a constant variable called todaysDate. Use method new Date; to define the variable as the current date. 
+console.log(todaysDate); //print todaysDate.
+
+//Write a JavaScript program to convert a number to a string.
+
+const num = 1982; //declare a constant variable called num and define it as 1982.
+const numToString = num.toString(); //declare a constant variable called numToString. use method toString to convert num to a string.
```

Comment:
this needs to be a function that takes in any number and returns the number converted as a string